source/jormungandr/tests/disruptions_tests.py

```diff
@@ -758,38 +758,99 @@ def test_disruption_with_stop_areas_and_different_parameters(self):
 
 @dataset({"line_sections_test": {}})
 class TestDisruptionsLineSections(AbstractTestFixture):
+    def _check_response(self, response):
+        line_reports = get_not_null(response, 'line_reports')
+        assert len(line_reports) == 2
+        # line:1
+        line_report = line_reports[0]
+        is_valid_line_report(line_report)
+        line = line_report['line']
+        assert line['id'] == 'line:1'
+        assert len(line['links']) == 2
+        assert line['links'][0]['id'] == 'line_section_on_line_1'
+        assert line['links'][1]['id'] == 'line_section_on_line_1_other_effect'
+        # line:1 associated pt_objects
+        pt_objects = line_report['pt_objects']
+        assert len(pt_objects) == 5
+        # C_1
+        assert pt_objects[0]['id'] == 'C_1'
+        assert len(pt_objects[0][pt_objects[0]['embedded_type']]['links']) == 1
+        link = pt_objects[0][pt_objects[0]['embedded_type']]['links'][0]
+        assert link['id'] == 'line_section_on_line_1'
```

Comment:
minor: shorter for 3 lines (can be done everywhere)
```python
assert [l['id'] for l in pt_objects[0][pt_objects[0]['embedded_type']]['links']] == ['line_section_on_line_1']
```
You may also use a set if you don't want to care about sorting.

Reply:
nice :+1: